sriov/tests/SR_IOV_BondVF_DPDK/test_SR_IOV_BondVF_DPDK.py

```diff
@@ -56,11 +56,15 @@ def dut_setup(dut, settings, testdata, request) -> Bond:
 
     execute_and_assert(dut, steps, 0, 0.1)
 
-    pci_pf1_vf0 = get_pci_address(dut, pf1 + "v0")
+    pci_pf1_vf0 = get_pci_address(
```

Comment:
Liked this change

Reply:
for history reason, we don't have an interface name for vf1 in the testbed.yaml, for handling consistency, maybe we leave it the way it was and clean this up later in a separate PR where we add the interface name for vf1